ocaml/xapi-idl/cluster/cluster_interface.ml

```diff
@@ -318,6 +359,12 @@ module LocalAPI (R : RPC) = struct
       @-> returning unit_p err
       )
 
+  let switch_cluster_stack =
+    let cluster_stack_p = Param.mk ~name:"cluster_stack" my_string in
+    declare "switch_cluster_stack_version"
```

Comment:
I think this string should match the function name, so no `_version` at the end. Also, the other ones above use hyphens. I think it is used in the CLI that is generated from this.

Reply:
Yes, good point. Fixed